src/encoded/static/components/cart/manager.js

```diff
@@ -706,31 +715,50 @@ const cartTableColumns = {
 /**
  * Display a count of the number of non-deleted carts, and the maximum possible number of carts.
  */
-const CartCounts = ({ cartManager }) => {
-    const cartCount = cartManager['@graph'].reduce((sum, cart) => (cart.status !== 'deleted' ? sum + 1 : sum), 0);
-    return (
-        <div className="cart-counts">
-            {cartCount} cart{cartCount !== 1 ? 's' : ''} ({cartManager.cart_user_max} maximum)
-        </div>
-    );
-};
+const CartCounts = ({ cartManager, cartCount }) => (
+    <div className="cart-counts">
+        {cartCount} cart{cartCount !== 1 ? 's' : ''} ({cartManager.cart_user_max} maximum)
+    </div>
+);
 
 CartCounts.propTypes = {
     /** cart-manager object from database */
     cartManager: PropTypes.object.isRequired,
+    /** Number of carts not deleted and not auto save */
```

Comment:
"saved"?

Reply:
I meant the proper noun “Auto Save cart,” though I should have capitalized.